client/dashboard/data/domains-dnssec.ts

```diff
@@ -0,0 +1,38 @@
+import wpcom from 'calypso/lib/wp';
```

Comment:
[nit] can we update the filename to `domain-dnssec.ts` to indicate that this is about one domain? For consistency with `site-*`.